Source/Source.php

```diff
@@ -141,7 +141,7 @@ function menu_main() {
 		$t_menu_options = array();
 
 		if ( plugin_config_get( 'show_repo_link' ) ) {
-			$t_page = plugin_page( 'index', false, 'Source' );
+			$t_page = plugin_page( 'index', true, 'Source' );
```

Comment:
>  I'll move it to another pull request. 

Thanks. I'll wait for you to do that, then we can follow up on this discussion there. 

The fact that it works for me as it is on several instances (including http://mantisbt.org/bugs) leads me to believe that it must be something related to your environment's setup, and I'd like to understand what it is.